gafffoyer/xml/gaff_nosmartsdefs.xml

```diff
@@ -1,4 +1,4 @@
-<ForceField>
+<ForceField name="GAFF2-nosmarts" version="0.0.1">
```

Comment:
Oops look like you flipped the `GAFF/GAFF2` names.

Reply:
oops! 🤦‍♀️